brightener.h

```diff
@@ -4,7 +4,7 @@
 struct Image {
 	int rows;
 	int columns;
-	uint8_t pixels[1024 * 1024]; // max 1k x 1k image
+	uint8_t* pixels = new uint8_t[1024 * 1024]; // max 1k x 1k image
```

Comment:
Good approach to place pixels on the heap, but we have a memory leak. There needs to be a `delete` corresponding to this?
You cannot do it in the destructor, since this instance is copied around. What can be a way out?